config/resources/domain.json

```diff
@@ -594,12 +634,40 @@
     },
     "central-worship-services": {
       "class": "ere:CentraalBestuurVanDeEredienst",
-      "super": ["worship-administrative-units"],
+      "super": [
+        "worship-administrative-units"
+      ],
       "new-resource-base": "http://data.lblod.info/id/centraleBesturenVanDeEredienst/"
     },
+    "kbo-administrative-units": {
```

Comment:
**Suggestion:** What rename `kbo-administrative-units` to `kbo-organisations` and use `ext:KboOrganization` as class?